cdn-examples/server/auth0/README.md

```diff
@@ -2,6 +2,26 @@
 
 This is an example of a Cloudflare worker that can be setup as a proxy with Auth0 integrations both injecting client side script and handling the verification on the CDN level. This worker would be setup to work as a proxy for Auth0 requests, on the GET based login screen for Auth0 this proxy would inject javascript into the page that would hijack the form submission and load an Arkose challenge on submission. On the POST based request this worker would intercept the request, extract the Arkose token from a definied cookie and verify the token. On successful verification the request would continue, on failed verification the request would redirect to a defined login page.
 
+## Required API
+This example utilises the Buffer API in Node.js. This requires some setup on the worker to use this API.
+There are multiple ways to do this:
+1. Via the `wrangler.toml` file.
+If you are managing your worker and deployments via Wrangler, and use `wrangler.toml` files, the best way to include the Buffer API is via setting the `node_compat` to `true`. I.e.
+```
+name = "auth0-dx-cloudflare-worker"
+main = "./index.js"
+compatibility_date = "2023-05-09"
+node_compat = true
+```
+Developer documentation found here: https://developers.cloudflare.com/workers/configuration/compatibility-dates/#nodejs-compatibility-flag
+
+2. Importing Buffer directly in the code
+Another method is to add an import by adding a line:
```

Comment:
Might be a good idea to specify where to add the line in our example specifically, just in case we have someone with low experience trying to set this up

Reply:
just realised i added this to the wrong readme...